doc/glm/glm_intro.rst

```diff
@@ -49,60 +83,108 @@ From the knowledge of the impulse haemodynamic response, we can build a predicte
+in the model: in practice, the number of observations (i.e. the number of time points), `n_scans` minus the number of effects
+modelled (i.e. the number of columns `n_columns`) of the design matrix:
 
-:math: `dof = n_scans - n_columns`
+ :math:`dof = n_scans - n_columns`
 
-With this we can do statistical inference: Given a pre-defined error rate :math:`\alpha`, we compare the observed `t` to the :math:`(1-\alpha)` quantile of the Student distribution with `dof` degrees of freedom. If t is greater than this number, we can reject the null hypothesis with a *p-value* :math:`\alpha`, meaning: if there were no effect, the probability of oberving an effect as large as `t` would be less than `\alpha`.
+With this we can do statistical inference. Given a pre-defined error rate :math:`\alpha`, we compare the observed `t` to
+the :math:`(1-\alpha)` quantile of the Student distribution with `dof` degrees of freedom. If `t` is greater than this number
+we can reject the null hypothesis with a *p-value* :math:`\alpha`; meaning, if there were no effect, the probability of
+observing an effect as large as `t` would be less than :math:`\alpha`.
 
 .. figure:: ../images/student.png
 
-.. note:: A frequent misconception consists in interpreting :math:`1-\alpha` as the probability that there is indeed an effect: this is not true ! Here we rely on a frequentist approach, that does not support Bayesian interpretation. See e.g. https://en.wikipedia.org/wiki/Frequentist_inference
-          
-.. note:: It is cumbersome to work with Student distributions, since those always require to specify the number `dof` of degrees of freedom. To avoid this, we can transform `t` to another variable `z` such that comparing `t` to the Student distribution with `dof` degrees of freedom is equivalent to comparing `z` to a standard normal distribution. We call this a z-transform of `t`. We call the :math:`(1-\alpha)` quantile of the normal distribution the *threshold*, since we use this value to declare voxels active or not.
-          
-Multiple Comparisons
---------------------
-
-A well-known issue that arrives then is that of multiple comparisons:
- when a statistical tests is repeated a large number times, say one for each voxel, i.e. `n_voxels` times, then one can expect that, in the absence of any effect,  the number of detections ---false detections since there is no effect--- will be roughly :math:`n\_voxels \alpha`. Then, take :math:`\alpha=.001` and :math:`n=10^5`, the number of false detections will be about 100. The danger is that one may no longer trust the detections, i.e. values of `z` larger than the :math:`(1-\alpha)`-quantile of the standard normal distribution.
+.. note:: A frequent misconception consists in interpreting `1- :math:\alpha` as the probability that there is indeed an
+effect: this is not true! Here we rely on a frequentist approach, that does not support Bayesian interpretation.
```

Comment:
Circleci complains about that line: "Explicit markup ends without a blank line; unexpected unindent."